libbeat/idxmgmt/ilm/config.go

```diff
@@ -103,20 +103,20 @@ func (m *Mode) Unpack(in string) error {
 
 //Validate verifies that expected config options are given and valid
 func (cfg *Config) Validate() error {
-	if cfg.RolloverAlias == "" && cfg.Mode != ModeDisabled {
+	if &cfg.RolloverAlias == nil && cfg.Mode != ModeDisabled {
 		return fmt.Errorf("rollover_alias must be set when ILM is not disabled")
 	}
 	return nil
 }
 
 func defaultConfig(info beat.Info) Config {
-	name := fmt.Sprintf("%s-%s", info.Beat, info.Version)
+	const name = "%{[beat.name]}-%{[beat.version]}"
```

Comment:
I think `beat.name` can be overwritten using the `name` setting. Also `beat.version` is deprecated and should be `agent.version`.

Proposal: `name := info.Beat + "-%{[agent.version]}"`